src/index.ts

```diff
@@ -1685,4 +1685,25 @@ export class MintSDK {
       }
     )
   }
+
+  /**
+   * 商品をid指定でアイテムを取得
+   * Returns the TokenERC721 from the specified tokenERC721Id.
+   *
+   * @param tokenERC721Id
+   * @returns
+   *
+   * ```typescript
+   * import { MintSDK } from '@kyuzan/mint-sdk-js'
+   * const sdk = new MintSDK(...)
+   * const item = await sdk.getTokenERC721ById('item.id')
```

Comment:
At Mint, `Item` has different meaning so plz fix
```diff
- const item = await sdk.getTokenERC721ById('item.id')
+ const item = await sdk.getTokenERC721ById('tokenERC721.id')
```

Reply:
@tomonari-t -san. I have changed `item` to `tokenERC721`. Please check it